### ISSUE TITLE
is it fast what I got for this file vfdt.py, do you have really big data sets to run?

### ISSUE THREAD

**Issue by Sandy4321** [doubleplusplus/incremental_decision_tree-CART-Random_Forest_python#1]
is it fast what I got for this file vfdt.py for windows 10 laptop
do you have really big data sets to run?

Total data size:  5000
Training size:  4500
Test set size:  500
Training set: 1000, ACCURACY: 0.7240
Training set: 2000, ACCURACY: 0.7500
Training set: 4500, ACCURACY: 0.7680
--- Running time: 0.936634 seconds ---

**Comment by doubleplusplus** [comment]
it's not fast on very large data sets. After all it's designed for continuous data, and produces results periodically.

**Comment by Sandy4321** [comment]
1
in any case it needs to try, may you share bit data set to try?
2
After all it's designed for continuous data
do you mean  that it will not work on categorical data?

**Comment by doubleplusplus** [comment]
There are some datasets in the dataset folder.

It works on categorical data.
